### PR TITLE
【NPU】add relu op for  npu

### PR DESCRIPTION
**Pull request by frankwhzhang**
### PR types

New features
### PR changes

OPs
### Describe

add relu npu

### CODE REVIEW

File: paddle/fluid/operators/activation_op_npu.cc [hunk 1]
```diff
@@ -103,6 +103,46 @@ class PowGradNPUKernel : public framework::OpKernel<T> {
...
+    auto* out = ctx.Output<Tensor>("Out");
+
+    out->mutable_data<T>(ctx.GetPlace());
+
+    auto runner = NpuOpRunner("Relu",
+                              {
+                                  *x,
+                              },
+                              {*out}, {});
+
```
Comment (zhiqiu) [inline paddle/fluid/operators/activation_op_npu.cc:120]: pre-commit

File: paddle/fluid/operators/activation_op_npu.cc [hunk 2]
```diff
@@ -103,6 +103,46 @@ class PowGradNPUKernel : public framework::OpKernel<T> {
...
+            .stream();
+    runner.Run(stream);
+  }
+};
+
+template <typename DeviceContext, typename T>
+class ReluGradNPUKernel : public framework::OpKernel<T> {
+ public:
+  void Compute(const framework::ExecutionContext& ctx) const override {
+    auto* x = ctx.Input<Tensor>("X");
```
Comment (zhiqiu) [inline paddle/fluid/operators/activation_op_npu.cc]: X -> Out

File: paddle/fluid/operators/activation_op_npu.cc (hunk 1 above)
Comment (frankwhzhang) [reply paddle/fluid/operators/activation_op_npu.cc:120]: this is after style

File: paddle/fluid/operators/activation_op_npu.cc (hunk 2 above)
Comment (frankwhzhang) [reply paddle/fluid/operators/activation_op_npu.cc]: fixed

**Comment by frankwhzhang** [comment]
屏幕快照 2021-03-12 下午1 50 59

**Review by zhiqiu — APPROVED** [review]
LGTM

**Pull request merged**